Cargo.toml

```diff
@@ -81,3 +81,6 @@ harness = false
 
 [package.metadata.docs.rs]
 all-features = true
+
+[patch.crates-io]
+backtrace = { git = 'https://github.com/hehechen/backtrace-rs' }
```

Comment:
It's not acceptable for pprof-rs to use a git dependency. Please publish it on crates.io :thinking:

Reply:
> It's not acceptable for pprof-rs to use a git dependency. Please publish it on crates.io 🤔

Done. https://crates.io/crates/pprof-backtrace